workflows/rnaseq/rnaseq-variant-calling.wdl

```diff
@@ -0,0 +1,132 @@
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+import "../../tools/picard.wdl"
+import "../../tools/gatk4.wdl" as gatk
+
+workflow rnaseq_variant_calling {
+    meta {
+        description: "Call variants from RNA-Seq data. Produces a VCF file of variants. Based on GATK RNA-Seq short variant calling best practices pipeline."
+        outputs: {
+            recalibrated_bam: "BAM that has undergone recalibration of base quality scores"
+            recalibrated_bam_index: "Index file for recalibrated BAM file"
+            merged_vcf: "VCF of raw RNA-Seq variant calls"
+            merged_vcf_index: "Index for merged VCF file"
+            variant_filtered_vcf: "VCF file after variant filters have been applied"
+            variant_filtered_vcf_index: "Index for filtered variant VCF file"
+        }
+    }
+
+    parameter_meta {
+        bam: "BAM file of aligned RNA-Seq reads"
+        bam_index: "Index file for BAM file"
+        fasta: "Reference FASTA file"
+        fasta_index: "Index file for reference FASTA file"
+        dict: "Sequence dictionary for reference FASTA file"
+        callingIntervalList: "Interval list of regions from which to call variants. Used for parallelization."
+        knownVcfs: "Array of known indels VCF files"
+        knownVcfIndexes: "Array of index files for known indels VCF files"
+        dbSNP_vcf: "dbSNP VCF file"
+        dbSNP_vcf_index: "Index file for dbSNP VCF file"
+        scatterCount: "Number of intervals to scatter over"
+        output_vcf_name: "Name of output VCF file"
+    }
+
+    input {
+        File bam
+        File bam_index
+        File fasta
+        File fasta_index
+        File dict
+        File callingIntervalList
+        Array[File] knownVcfs
+        Array[File] knownVcfIndexes
+        File dbSNP_vcf
+        File dbSNP_vcf_index
+        Int scatterCount = 6
+        String output_vcf_name = basename(bam, '.bam') + '.vcf.gz'
+    }
+    
+    call picard.mark_duplicates {
+        input: 
+            bam = bam,
+            create_bam = true
+    }
+
+    call gatk.split_n_cigar_reads {
+        input:
+            bam = select_first([mark_duplicates.duplicate_marked_bam, '']),
```

Comment:
I think inconsistency on this is _ok_, so won't hold up the PR on this change. But I think the literal in a select_first should be `"undefined"`. That's clearer to me than an empty string. Also ok with `error` or something like that. I just like it to be more explicit.

Also we've switched to using double quotes everywhere. Please double check if I missed any single quotes elsewhere.